apps/api_web/lib/api_web/controllers/vehicle_controller.ex

```diff
@@ -244,6 +244,25 @@ defmodule ApiWeb.VehicleController do
               "Speed that the vehicle is traveling in meters per second. See [GTFS-realtime Position speed](https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/reference.md#message-position).",
               example: 16
             )
+
+            occupancy_status(
+              :string,
```

Comment:
Can we mark this as `nullable` too?

Reply:
yup, actually the failing test is due to me not marking it as nullable

Reply:
sorry, squashed and force pushed before i saw this, so this change is in the initial commit